File: practice-onet/get.py
import requests

def fetch_onet_data(onet_code);
    onet_url = f"https://services.onetcenter.org/ws/mnm/careers/{onet_code}/summary"
    response = requests.get(onet_url)
    if response.status_code == 200:
        return response.json()
    else:
        return None

if __name__ == "__main__":
    onet_code = "151131.00" # Example O*NET code for "Software Developers, Applications"
    onet_code = fetch_onet_data(onet_code)
    if onet_data:
        print(onet_data)
    else:
        print("Failed to fetch data from O*NET")

File: practice-onet/get.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.net.HttpURLConnection;
import java.net.URL;

public class OnetCodeFetcher {

    public static void main(String[] args) throws Exception {
        String onetCode = "151131.00"; // Example 0*NET code for Software Developers, Applications"
        String onetUrl = "https://services.onetcenter.org/ws/mnm/careers/" + onetCode + "/summary";

        URL url = new URL(onetUrl);
        HttpURLConnection conn = (HttpURLConnection) url.openConnection();
        conn.setRequestMethod("GET");

        BufferedReader in = new BufferedReader(new InputStreamReader(conn.getInputStream()));
        String inputLine;
        StringBuffer response = new StringBuffer();

        while ((inputLine = in.readLine()) != null) {
            response.append(inputLine);
        }
        in.close();

        System.out.println(response.toString());
    }
}